assets/scss/app.scss

```diff
@@ -314,3 +314,12 @@ textarea {
   white-space: pre-wrap;
   word-wrap: break-word;
 }
+
+.form-control-100 {
+  width: 100%;
+}
```

Comment:
ah ok now I see why you did this. Does this impact mobile devices?

Reply:
no, it just adjusts with all the other fields, I gave it a test in Chrome. Just makes the text areas slightly wider